doc/usage/configuration.rst

```diff
@@ -520,25 +520,26 @@ Project information
    .. versionadded:: 0.5
 
    .. versionchanged:: 1.4
-      The default is now ``'default'``. It is similar to ``'python3'``;
-      it is mostly a superset of ``'python'``. but it fallbacks to
-      ``'none'`` without warning if failed.  ``'python3'`` and other
-      languages will emit warning if failed.  If you prefer Python 2
-      only highlighting, you can set it back to ``'python'``.
+      The default is now ``'default'``. It is similar to ``'python3'``; it is
+      mostly a superset of ``'python'``. but it fallbacks to ``'none'`` without
+      warning if failed.  ``'python3'`` and other languages will emit warning
+      if failed.  If you prefer Python 2 only highlighting, you can set it back
+      to ``'python'``.
 
 .. confval:: highlight_options
 
    A dictionary of options that modify how the lexer specified by
    :confval:`highlight_language` generates highlighted source code. These are
-   lexer-specific; for the options understood by each, see the
-   `Pygments documentation <http://pygments.org/docs/lexers/>`_.
+   lexer-specific; for the options understood by each, see the `Pygments
+   documentation <http://pygments.org/docs/lexers/>`_.
```

Comment:
is doc8 mandating that the link stand on a line of its own? but reStructured Text allows the linebreak.

Reply:
oops, here actually the original had the link on one line in source...